tests/bugs/buffer-swizzle-store.slang

```diff
@@ -1,7 +1,7 @@
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute -shaderobj -output-using-type
 //TEST(compute, vulkan):COMPARE_COMPUTE_EX:-vk -compute -shaderobj -output-using-type
 //TEST(compute):COMPARE_COMPUTE:-slang -shaderobj -mtl -output-using-type
-// Slang-RHI/WGPU: Invalid bind group layout entry used #5612
+// Not supported in WGSL: read-write storage texture with "rg16f" format
```

Comment:
If the format is only problem for WGSL, can we change the format to one of formats WGSL supports?
Or maybe we can copy this test with a different format.

Reply:
Yeah I can give it a shot.
There are [some Slang-RHI issues related to storage textures](https://github.com/shader-slang/slang/issues/5612#issuecomment-2514257576) I need to fix first though -- it currently doesn't create the bind group layout entry properly. I think I should set the `access` field to `read-write`, but I'm not quite sure how I should then get the required format information. I think I'll have to check in with Simon.